docs/conf.py

```diff
@@ -153,13 +153,20 @@
             "icon": "fa-brands fa-discord",
         },
     ],
+    "external_links": [
+        {
+            "name": "Survey",
+            "url": "https://docs.google.com/forms/d/e/1FAIpQLSeOTRNgAc2RYnDDO5-GE7scMysPHrBLs8XV21ZP--XATr34aA/viewform",
```

Comment:
I forgot we have a shorter link, would you mind replacing it everywhere: https://forms.gle/U3wdiNAPeQWgenKh8

It will make for clearer reading of the code.

Reply:
Changed all links to shorter link